docs/api/segment.md

```diff
@@ -49,6 +49,20 @@ import Toolbar from '@site/static/usage/v8/toolbar/segments/index.md';
 <Toolbar />
 
 
+## Swipeable Segments
+
+Each [segment button](./segment-button.md) can be associated with a [segment content](./segment-content.md) element that will be displayed
+when the segment is active. With this approach, each segment's content can be swiped or scrolled between and the segment will be updated
+to reflect the currently visible content.
+
+If no initial `value` is assigned to the `ion-segment` when using swipeable segment, the segment will default to the value of the first segment button.
+
+Segment buttons cannot be disabled when used with swipeable segments.
```

Comment:
I wonder if this should be a warning admonition. I'll leave it up to you, I'm good either or.